lib/CMakeLists.txt

```diff
@@ -161,11 +195,6 @@ else()
 	target_link_libraries(chiaki-lib OpenSSL::Crypto)
 endif()
 
-if (CHIAKI_USE_SYSTEM_CURL)
```

Comment:
if curl comes after mbedtls, it fails at linking time as it is unable to find the mbedtls references.